docs/src/catalyst_for_new_julia_users/intro_to_catalyst.md

```diff
@@ -0,0 +1,182 @@
+
+### Installing and activating packages
+Except for some base Julia packages (such as `Pkg`, the package manager) that are available by default, Julia packages must be installed locally before they can be used. Most packages are registered with Julia, and can be added through the `Pkg.add("desired_package")` command (where `desired_package` is the name of the package you wish to install). We can thus install Catalyst:
+```julia
+using Pkg
+Pkg.add("Catalyst")
+```
+
+Here, the command `using Pkg` is required to activate the package manager.
+
+Next, we also wish to add the `DifferentialEquations` and `Plots` packages (for numeric simulation of models, and plotting, respectively). 
+```julia
+Pkg.add("DifferentialEquations")
+Pkg.add("Plots")
+```
+Once a package has been installed through the `Pkg.add` command, this command does not have to be repeated in further Julia sessions on the same machine.
+
+Installing a Julia package is, however, not enough to use it. Before a package's features are used in a Julia session, it has to be activated through the `using desired_package` command (where `desired_package` is the name of the package you wish to activate). This command has to be repeated whenever a Julia session is restarted.
+
+We thus activate our three desired packages:
+
+```julia
+using Catalyst
+using DifferentialEquations
+using Plots
+```
+
+For a more detailed introduction to Julia packages, please read [the Pkg documentation](https://docs.julialang.org/en/v1/stdlib/Pkg/).
+
+### Simulating a basic Catalyst model
+Now that we have some basic familiarity with Julia, and have installed and activated the required packages, we will create and simulate a basic chemical reaction network model through Catalyst.
+
+Catalyst models are created through the `@reaction_network` *macro*. For more information on macros, please read [the Julia documentation on macros](https://docs.julialang.org/en/v1/manual/metaprogramming/#man-macros). This documentation is, however, rather advanced (and not required to use Catalyst). We instead recommend that you simply familiarise yourself with the Catalyst syntax, without studying in detail how macros work and what they are.
+
+The `@reaction_network` command is followed by the `begin` keyword, which is followed by one line for each *reaction* of the model. Each reaction consists of a *reaction rate*, followed by the reaction itself. The reaction itself contains a set of *substrates* and a set of *products* (what is consumed and produced by the reaction, respectively). These are separated by a `-->` arrow. Finally, the model ends with the `end` keyword, followed by a list of its *parameters*.
+
+Here, we create a simple *birth-death* model, where a single species (*X*) is created at rate *b*, and degraded at rate *d*. The model is stored in the variable `rn`.
+
+```julia
+rn = @reaction_network begin
+    b, 0 --> X
+    d, X --> 0
+end b d;
```

Comment:
Drop the `;` to show the output?

Reply:
For some reason I got an error when I tried to display the network on my machine, so added this one here. Generally, I had several `;` to avoid clogging up the docs. But adding this one back in makes sense, and maybe I should do that for more blocks.